action.yml

```
@@ -1,14 +1,17 @@
-name: 'Release prep'
-description: 'Common setup for craft/prepare'
+name: "Release prep"
```

_nit_: Since this is a title that shows up in every repository, it would be nice if this is not abbreviated (and title-case):

```suggestion
name: "Prepare Release"
```